src/opm/output/eclipse/Summary.cpp

```diff
@@ -1430,26 +1430,25 @@ well_efficiency_factors( const ecl::smspec_node* node,
 
     const bool is_group = (var_type == ECL_SMSPEC_GROUP_VAR);
     const bool is_rate = !node->is_total();
-    const auto &groupTree = schedule.getGroupTree(sim_step);
+    auto groupTree = schedule.groupTree(sim_step);
```

Comment:
Do you need to mutate `groupTree`?  If not, please declare it `const`.

Reply:
Personally I think `const` on local value looks funny; but anyway I will add the `const`